### DISCUSSION TITLE
启动器停止正常工作

### DISCUSSION THREAD

**Discussion by beibei-b** [Meloong-Git/PCL#2084] [提问]
我正常安装pcl2，双击打开应用程序后弹出六边形的PCL图样（到这都是正常的）
但图样消失后并没有出现熟悉的操作界面，取而代之的是一个“PCL2停止工作”的窗口
启动器版本2.2.1和2.6.1都尝试了一下，一样的
图2为启动器日志文件（应该吧，不懂
![pcl error](https://github.com/Hex-Dragon/PCL2/assets/137275723/5a0720a2-8e6e-402e-83b0-19c256d015e9)
![pcl error2](https://github.com/Hex-Dragon/PCL2/assets/137275723/0d7a5e62-53fb-4dab-b04d-8c381f1e67ec)

**Comment by Pigeon0v0** [comment]
安装 .NET Framework 4.6.2 或以上版本

**Reply by beibei-b** [reply]
好像没用，这是4.8.0的呀
![netframework4 8 0](https://github.com/Hex-Dragon/PCL2/assets/137275723/54dfe191-0f1b-4a73-9678-ae4e982fee09)

**Comment by Pigeon0v0** [comment]
不对，Log 都出来了…

上传 Log 文件看看

**Comment by LTCatt** [comment]
先更新，发最新版的 log

**Reply by beibei-b** [reply]
启动器版本：2.6.1，游戏版本：fabric-loader-0.14.9-1.18.2，登录方式：微软登录
另外，官方启动器和hmcl都能正常使用（可我还是习惯用pcl）
![pcl error4](https://github.com/Hex-Dragon/PCL2/assets/137275723/f549fee9-42d6-485b-8923-19621fb559ff)

**Reply by LTCatt** [reply]
发文件啊求求了

**Reply by beibei-b** [reply]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/11833034/Log1.txt)

**Reply by LTCatt** [reply]
看不出来有啥问题……这个我没办法了 orz

**Reply by beibei-b** [reply]
这是我安装pcl2的视频，qq录屏录的，是不是安装时出了什么问题

https://github.com/Hex-Dragon/PCL2/assets/137275723/6d4fa5eb-09f4-4adf-8900-f9c80389f624

**Reply by beibei-b** [reply]
已安装最新版，问题依然存在
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/11932495/Log1.txt)
